pkg/controllers/util_api.go

```diff
@@ -0,0 +1,33 @@
+package controllers
+
+import (
+	apiv1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	appslisters "k8s.io/client-go/listers/apps/v1beta1"
+)
+
+func PodControlledByCluster(
```

Comment:
Can we also update the Elasticsearch controller to use this function too? This version doesn't depend on any types like *ElasticsearchCluster like the one the the controller currently uses

Reply:
Done.